modules/gltf/gltf_document.cpp

```diff
@@ -403,6 +410,15 @@ Error GLTFDocument::_serialize_nodes(Ref<GLTFState> state) {
 			}
 			node["children"] = children;
 		}
+
+		for (int32_t ext_i = 0; ext_i < document_extensions.size(); ext_i++) {
+			Ref<GLTFDocumentExtension> ext = document_extensions[ext_i];
+			ERR_CONTINUE(ext.is_null());
+			ERR_CONTINUE(i >= state->scene_nodes.size());
+			Error err = ext->export_node(state, n, node, state->scene_nodes[i]);
```

Comment:
`export_node` is also called in `append_from_scene`

Reply:
Any progress on this issue?

Reply:
Ok, it seems fine now.

I don't want to discourage you from having both if there's a reason for it, but there needs to be a way to distinguish the two calls if so, for example the InternalImportCategory.